### ISSUE TITLE
Investigate extracted FDA dates

### ISSUE THREAD

**Issue by nightsh** [opentrials/opentrials#562]
Filing a quick issue for an unexpected find, to be investigated.

In my local database, which was dumped from production a week ago, I have the following FDA approval JSON:

```json
      "fda_approval": {
        "id": "NDA021945-000",
        "supplement_number": 0,
        "type": "Approval",
        "action_date": "2011-02-02T22:00:00.000Z",
        "fda_application": {
          "id": "NDA021945",
          "drug_name": "MAKENA",
          "active_ingredients": "HYDROXYPROGESTERONE CAPROATE",
          "url": "http://localhost:10010/v1/fda_applications/NDA021945",
          "type": "NDA"
        }
      },
```

Now go to http://www.accessdata.fda.gov/scripts/cder/daf/index.cfm -> search for MAKENA -> click on the first result -> click on the approval history link. The action date here is `02/03/2011`.

Not sure it's an issue, but it's probably worth checking.

**Comment by vitorbaptista** [comment]
This is probably caused by timezones. My hypothesis is that, when we're parsing the date in the collector, it considers that it's in the current machine's timezone (e.g. UTC-2). However, when saving it or displaying it, it converts it to UTC, and so there's the difference we see.

**Comment by nightsh** [comment]
That's also my assumption so far. I haven't investigated it yet.

**Comment by georgiana-b** [comment]
Closed in favor of #586 because the scope is broader than FDA.
